### DISCUSSION TITLE
Options in Context menu

### DISCUSSION THREAD

**Discussion by IkelAtomig** [coddingtonbear/obsidian-local-rest-api#51] [Ideas]
I use the extension mostly for saving content. It would be great if options in Dropdown menu is available in context menu. Instead of going to the extension back and forth which is located in Toolbar.

**Comment by coddingtonbear** [comment]
You probably intend to post this to https://github.com/coddingtonbear/obsidian-web 's discussions (and I believe there's already a similar one).  This repository covers only the API, not the interface you're using for interacting with obsidian.
